### PR TITLE
Spam Reasons translateable

### PR DESCRIPTION
**Pull request by websupporter**
Solves #82 

The spam reasons are translateable now and I quickly changes the name "RegExp" to "Regular Expression".

### CODE REVIEW

File: antispam_bee.php [hunk 1]
```diff
@@ -396,7 +396,7 @@ private static function _init_internal_vars()
 				'dnsbl'		=> __( 'DNSBL Spam', 'antispam-bee' ),
```
Comment (schlessera) [inline antispam_bee.php]: If you are already making changes to the names, the DNSBL was an old spam database that we don't use anymore. This now uses StopForumSpam database. Maybe we should even change the key `dnsbl` and the method names to keep it consistent, not only the string.

Comment (schlessera) [inline antispam_bee.php]: Hmm, but maybe in a different PR...

Comment (websupporter) [inline antispam_bee.php]: Could have been seperated. I agree. When we change the key, we get a problem with old data, where the spam reason is saved as `dnsbl`, which will not be recongnized no more, when I see this correctly?

Comment (schlessera) [inline antispam_bee.php]: Yes, we need a migration path for spam reasons, if they evolve like that

**Comment by websupporter** [comment]
For now, I use a more generic term for DNSBL: `Public Antispam DB`, its already quite long, so I didn't wrote `database`.

**Comment by websupporter** [comment]
@schlessera maybe you can have a look.

1. Why did it fail :laughing: 
2. Is this the right place for stub functions?

**Comment by schlessera** [comment]
Was an unrelated error with `rake` build failing. I restarted the check to see if it happens again...

**Comment by websupporter** [comment]
Maybe its because the branch does not have the yaml, since this file was introduced later?

**Comment by schlessera** [comment]
Yes, that might be the cause. Try rebasing the branch.

**Pull request merged**